contributions/README.md

```diff
@@ -0,0 +1,81 @@
+# Users
+
+## Contributions Object
+
+```
+{
+  prList: [{
+    'title': <title of pull request>,
+    'url': <url of pull request>,
+    'state': <open or closed>,
+    'createdAt': <unix_timestamp>,
+    'updatedAt': <unix_timestamp>,
+    'raisedBy': <username of the assignee>
+  }],
+  task: {
+    'title': <task tile>,
+    'purpose': <why this task is needed>,
+    'endsOn': <unix_timestamp>,
+    'startedOn': <unix_timestamp>,
+    'deployedOn': <unix_timestamp>,
+    'participants': [
+      {
+        'first_name': <first name of user>,
```

Comment:
Mix of snake_case and camelCase...
Can we stick to one convention?

Reply:
Sure !